### ISSUE TITLE
imap4 examine() and UNSEEN

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3550]
|<img alt="imgrey's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| imgrey reported|
|-|-|
|Trac ID|trac#3550|
|Type|defect|
|Created|2008-11-19 14:43:43Z|
|Branch|https://github.com/twisted/twisted/tree/examine-unseen-3550|

imap4.IMAP4Client.examine() wont return UNSEEN key

<details><summary>Searchable metadata</summary>

```
trac-id__3550 3550
type__defect defect
reporter__imgrey imgrey
priority__normal normal
milestone__ 
branch__branches_examine_unseen_3550 branches/examine-unseen-3550
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__mail mail
keywords__ 
time__1227105823000000 1227105823000000
changetime__1227132525000000 1227132525000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun commented|
|-|-|

(In [25440]) Branching to 'examine-unseen-3550'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun removed owner|
|-|-|

Thanks for the report!

I think this is fixed in the branch.

**Comment by twisted-trac** [comment]
|[<img alt="therve's avatar" src="https://avatars.githubusercontent.com/u/159137?s=50" width="50" height="50">](https://github.com/therve)<a name="note_3"></a>|@therve set owner to @exarkun|
|-|-|

Looks great, please merge.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun set status to `closed`|
|-|-|

(In [25459]) Merge examine-unseen-3550

Author: exarkun
Reviewer: therve
Fixes: [#3550](https://github.com/twisted/twisted/issues/3550)

Fix parsing of UNSEEN response data so that it is actually included in
the dictionary the Deferred returned by `IMAP4Client.examine` fires
with.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
